internal/api/utils.go

```diff
@@ -18,7 +18,9 @@ func getQueryBuilder(r *http.Request) (query.Builder, error) {
 	if len(data) > 0 {
 		return query.ParseJSON(string(data))
 	}
-	return nil, nil
+
+	// If we don't have a body, we use the query param
+	return query.ParseJSON(r.URL.Query().Get("query"))
```

Comment:
This will return errors on requests that don't provide any filters. I'm not sure if that's what you want?

You'll probably have to add some handling in the caller of this function that is aware of a nil QueryBuilder and doesn't append that to the req (eg. when we make requests with cursors)

I just implemented this in another service recently, so speaking from recent experience...